testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BokkenMachine.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Diagnostics;
+using System.ComponentModel;
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class BokkenMachine
+    {
+        public DateTimeOffset DateCreated { get; set; }
+        public string Name { get; private set; }
+        public string Type { get; set; }
+        public string Image { get; set; }
+        public string Flavor { get; set; }
+        public string JobID { get; set; }
+        public FileInfo JobFile { get; private set; }
+        public string PathToJson { get; set; }
+        public string LogPath { get; set; }
+
+        public Dictionary<string, BokkenMachine> BokkenMachines;
+
+        private static FileInfo s_FileInfo;
+        private static string s_Rootdir;
+        public static string PathToDll { get; private set; }
+
+        private static List<Process> s_ProcessList = new List<Process>();
+
+        static BokkenMachine()
+        {
+            s_FileInfo = new FileInfo(Path.Combine(MultiprocessOrchestration.MultiprocessDirInfo.FullName, "rootdir"));
+            s_Rootdir = (File.ReadAllText(s_FileInfo.FullName)).Trim();
+            PathToDll = Path.Combine(s_Rootdir, "BokkenForNetcode", "ProvisionBokkenMachines", "bin", "Debug", "netcoreapp3.1", "ProvisionBokkenMachines.dll");
+            MultiprocessLogger.Log($"Bokken Machine vars are: \n\ts_FileInfo {s_FileInfo.FullName}, {s_FileInfo.Exists} \n\ts_Rootdir {s_Rootdir}\n\tPathToDll {PathToDll}");
+        }
+
+        public static BokkenMachine GetDefaultMac(string name)
+        {
+            var rv = new BokkenMachine();
+            rv.Type = "Unity::VM::osx";
+            rv.Image = "unity-ci/macos-10.15-dotnetcore:latest";
+            rv.Flavor = "b1.large";
+            rv.Name = name;
+            return rv;
+        }
+
+        public static BokkenMachine GetDefaultWin(string name)
+        {
+            var rv = new BokkenMachine();
+            rv.Type = "Unity::VM";
+            rv.Image = "package-ci/win10:stable";
+            rv.Flavor = "b1.large";
+            rv.Name = name;
+            return rv;
+        }
+
+        public static BokkenMachine Parse(string shortcut)
+        {
+            BokkenMachine rv = null;
+            string[] parts = shortcut.Split(':');
+            string name = parts[1];
+            string type = parts[0];
+            if (type.Equals("default-mac"))
+            {
+                rv = GetDefaultMac(name);
+            }
+            else if (type.Equals("default-win"))
+            {
+                rv = GetDefaultWin(name);
+            }
+
+            return rv;
+        }
+        public BokkenMachine()
+        {
+        }
+
+        public void Provision()
+        {
+            ExecuteCommand(GenerateCreateCommand(), true, false, 5000);
```

Comment:
5000 == magic number

Reply:
Not sure what to do about timeouts.

Reply:
by magic number I meant it was a clarity issue.
https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Okay, I annotated this in the API to indicate that it is a timeout value and set a default. As per @NoelStephensUnity comment, this can be made configurable via command line parameters as well. @SamuelBellomo

Reply:
not sure I see the changes you describe, did you submit them? Maybe I missed them